From 6c3759e342efa8b9bcc46e1701eed3a12548db85 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Mon, 27 May 2024 04:59:20 +0000
Subject: [PATCH] shiboken: add missing include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Amend b829abcc7b2e9fcdb027e653a6a52cdb0706de11 and add missing include
to fix:

 | sbkmodule.cpp:459:15: error: ‘Shiboken::Conversions’ has not been declared
 |     Shiboken::Conversions::clearNegativeLazyCache();
 |               ^~~~~~~~~~~

Pick-to: 6.7
Fixes: PYSIDE-2765
Task-number: PYSIDE-2749
Task-number: PYSIDE-2404
Change-Id: I13625848bff3b0e90a53649104d6e5d84553a984
Reviewed-by: Friedemann Kleint <Friedemann.Kleint@qt.io>
---
 sources/shiboken6/libshiboken/sbkmodule.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sources/shiboken6/libshiboken/sbkmodule.cpp b/sources/shiboken6/libshiboken/sbkmodule.cpp
index ccc7cc2cd..b7e6bf126 100644
--- a/sources/shiboken6/libshiboken/sbkmodule.cpp
+++ b/sources/shiboken6/libshiboken/sbkmodule.cpp
@@ -7,6 +7,7 @@
 #include "bindingmanager.h"
 #include "sbkstring.h"
 #include "sbkcppstring.h"
+#include "sbkconverter_p.h"
 
 #include <unordered_map>
 #include <unordered_set>
-- 
2.45.1

